src/mango/src/mango_idx.erl

```diff
@@ -410,12 +421,20 @@ get_idx_partitioned(Db, DDocProps) ->
             Default
     end.
 
+is_opts_partitioned(Opts) ->
+    case couch_util:get_value(partition, Opts) of
```

Comment:
This call here makes a brave assumption that `partition` attribute is always presented in `Opts` (which might be true, I don't know), but I think `couch_util:get_value(partition, Opts, <<>>)` might be a bit safer here, just to avoid possibility of a crash on no case clause matching.